ci/deploy_mz-debug/README.md

```diff
@@ -22,13 +22,15 @@ You can manually deploy by following steps 1-2 above and running the following c
 ```bash
 # Set a tag version.
 export BUILDKITE_TAG=mz-debug-vx.y.z
+export AWS_PROFILE=...
 
 # macOS
 bin/pyactivate -m ci.deploy_mz-debug.macos
 
 # Linux
 bin/pyactivate -m ci.deploy_mz-debug.linux
 ```
+where AWS_PROFILE is the profile with access to the materialize-binaries S3 bucket in the Materialize Core account.
```

Comment:
I don't think devs have access to the Materialize Core account normally? At least I don't.

Reply:
What I always used to use is https://debuginfo.dev.materialize.com/buildid/d6a8b86c62ce2b7a6fd146a048ebba77/executable, which is publicly available. I think that's easier instead of using S3.